src/main/java/io/reactivex/Flowable.java

```diff
@@ -194,7 +194,7 @@ public static int bufferSize() {
      */
     @SchedulerSupport(SchedulerSupport.NONE)
     @BackpressureSupport(BackpressureKind.FULL)
-    public static <T, R> Flowable<R> combineLatest(Function<? super T[], ? extends R> combiner, Publisher<? extends T>... sources) {
+    public static <T, R> Flowable<R> combineLatest(Function<? super Object[], ? extends R> combiner, Publisher<? extends T>... sources) {
```

Comment:
this one does not seem necessary to be honest. It just has switched arguments with the one above. Or do I oversee anything?